File: ui/src/entity/message.js
class Message{
    constructor(kafkaCommand, topicName, contents, key, accessToken){
        this.kafkaCommand = kafkaCommand
        this.topicName = topicName
        this.contents = contents
        this.key = key
        this.accessToken = accessToken
    }
}

module.exports = {
    Message
}

File: safka/SafkaConsumer/src/main/java/org/alexstk/safka/consumer/entity/Message.java
package org.alexstk.safka.consumer.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NonNull;
import lombok.ToString;

import java.util.Random;

@Getter
@ToString
@AllArgsConstructor
public class Message {
    private String key;
    @NonNull
    private String kafkaCommand;
    @NonNull
    private String topicName;
    private String contents;


}
